Fix The LED order of wndr4500v3 devices to match case label numbers

moltencrux opened the pull request:
The LED and LAN port numbering on the case are reversed relative to the wndr4300v2.  I created this patch to so that the ordering in OpenWRT will be consistent with that.

mans0n commented:
Hi, this has some formal issues, especially:
* Please add a prefix "ath79:" to the commit title.
* Please move the PR description to the commit message.
* Please add a Signed-off-by tag with your real name and a reachable mail address.
https://openwrt.org/docs/guide-developer/frequent-pr-mistakes

moltencrux commented:
Amended the commit.  Hopefully I followed the guidelines correctly this time.

mans0n commented:
It sounds weird to me that the vendor used the same board but different port layout, so I searched for the photos and it turned out that you are right. xD
https://www.ebay.com/itm/133789322335
https://www.ebay.com/itm/304341926767

If the port numbers and the label numbers match, then we can safely skip the label numbers, so I would prefer `"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan" "5:wan"`.

While at it, could you also convert the commit title to all lowercase? (i.e. `Fix` -> `fix`) It would be better if you can also reduce the title up to 50 characters.

And the author line in the patch still uses noreply.github address, you can change it by `git config user.email <mail-address> && git commit --amend --reset-author`, or I can handle it myself if you are not using git CLI.

moltencrux committed:
ath79: fix wndr4500v3 lan port order to match case

The LED and LAN port numbering on the case of wndr4500v3 devices are
reversed relative to the wndr4300v2. I created this patch to so that the
ordering in OpenWRT will be consistent with that.

Signed-off-by: Graham Cole <diakka@gmail.com>

moltencrux commented:
I've amended the commit, hopefully third time is the charm :D

> EDIT: But they still share the same port order (LAN1 ... LAN4 WAN) so I'm still not 100% sure...

I own both devices, so I can confirm that the labeling is reversed.  The port labels and link lights on the wndr4300v2 were ordered correctly in LuCI before these changes.  Previously on the wndr4500v3 the LEDs that were lit did not correspond to the links in LuCI, but after the changing the script and generating a new board.json, the LuCI display matches the LED link lights.

mans0n commented:
> I own both devices, so I can confirm that the labeling is reversed.

Okay, that makes me feel more relieved. :)

This patch is actually changing the LAN port order and not LED order, so I'm thinking of rephrasing the title to something like `ath79: fix switch port order on WNDR4500 v3` before the merge. Or you can do it yourself if you're faster than me. ;)

moltencrux commented:
How's that?

mans0n commented:
Thanks! Merged your patch as e282e50d4491.
